quiz.js

```diff
@@ -0,0 +1,72 @@
+
+
+
+/*---- GET / TEST THE INPUT ------ */
+
+// function testInput(heightInput, charInput) {
+// 	console.log("It's working! ", heightInput, charInput);
+// 	console.log(input.height, input.character)
+// }
+
+function checkInput () {
+	var heightInput = document.getElementById("heightInput").value;
+	var charInput = document.getElementById("charInput").value;
+	
+	 if (heightInput && charInput){
+
+		beautifulTree(heightInput, charInput);
+	
+	 } else {
+			
+			alert("Please enter a height & character.")
+	 }
+}
+
+submit.addEventListener("click", checkInput, charInput);
+
+/*------------------------*/
+
+
+
+function beautifulTree (heightInput, charInput) {
+
+for (var i = 0; i < (heightInput * 2); i++) {
+	
+	var prettyTree = "";
+	prettyTree = charInput.repeat(i += 1);
+	
+	// var prettySpaces = "s";
+	// prettySpaces = prettySpaces.repeat(heightInput - i);
+
+	console.log(prettyTree);
+
```

Comment:
This logic is working great. You just need to add in a way to put a certain number of spaces in front of the characters each time through the loop. You're on the right track with calculating spaces as height minus one for the first line. Then you just keep subtracting one from the previous number of spaces each time through the loop